Lesson4.1/src/com/home/model/Computer.java

```diff
@@ -0,0 +1,97 @@
+package com.home.model;
+
+import com.home.service.Reader;
+
+import java.util.Random;
+
+public class Computer {
+
+    private String cpu;
+    private String ram;
+    private String hardDisk;
+    private int lifeCycle;
+
+    public Computer(String cpu, String ram, String hardDisk, int lifeCycle) {
+        this.cpu = cpu;
+        this.ram = ram;
+        this.hardDisk = hardDisk;
+        this.lifeCycle = lifeCycle;
+    }
+
+    public void info() {
+        System.out.println("Model of CPU: " + cpu);
+        System.out.println("Model of RAM: " + ram);
+        System.out.println("Model of hard disk: " + hardDisk);
+        System.out.println("Number of life cycle: " + lifeCycle);
+    }
+
+    public int turnOn(int lifeCycle) {
+        //int cycle = lifeCycle;
+        System.out.println("PC is turning on...");
+        if (getRandomNumber()) {
+            lifeCycle = turnOff(lifeCycle);
+            if (lifeCycle == 0) {
```

Comment:
You don't need to do a random check in this case, because a computer is already dead

Reply:
So turnOn() method shouldn't include Random? When pc is turning on at 1st time i should have random number checking.

Reply:
right, you need random checks while PC can be turned on(it didn't burn out because of failed check or because it's out of cycles). But you need to track if PC is still can be turned on. And if it's already burned out then you don't do this checks